lib/twilio/audiohelper.ts

```diff
@@ -424,12 +424,14 @@ class AudioHelper extends EventEmitter {
   }
 
   /**
-   * Adds an {@link AudioProcessor} object.
-   * The AudioHelper will route the input audio stream through the processor
-   * before sending the audio stream to Twilio.
+   * Adds an {@link AudioProcessor} object. Once added, the AudioHelper will route
+   * the input audio stream through the processor before sending the audio
+   * stream to Twilio. Only one AudioProcessor can be added at this time.
    *
-   * Only one {@link AudioProcessor} can be added at this time.
-   * @param processor
+   * See the {@link AudioProcessor} interface for an example.
+   *
+   * @param processor The AudioProcessor to add.
+   * @returns
```

Comment:
Should we specify what we return here? "A `Promise` that resolves..."

Reply:
This is already shown in the method signature.